misc/helm-charts/operator/README.md

```diff
@@ -295,11 +295,11 @@ Or check the `Chart.yaml` file in the `misc/helm-charts/operator` directory:
 apiVersion: v2
 name: materialize-operator
 # ...
-version: v25.2.0-beta-1
-appVersion: v0.130.3  # Use this version for your Materialize instances
+version: v25.3.0-beta-1
+appVersion: v0.147.1  # Use this version for your Materialize instances
```

Comment:
Mentioned on slack, but I think I mislead you and we might be able to use 0.147.0 here since any backports were only for the operator not for the database